src/app/pages/clients/clients.component.html

```diff
@@ -0,0 +1,61 @@
+<div>
+  <div class="row">
+    <div class="col-md-8">
+      <div class="row">
+        <div class="col-md-6" *ngFor="let client of clients">
+          <div class="card listview">
+            <div class="row no-gutters">
+              <div class="col">
+                <div class="card-block px-2">
+                  <h4>
+                    <b class="card-title"> {{ client.client_name }}</b>
+                  </h4>
+
+                  <ul class="list-unstyled fa-ul">
+                    <li>
+                      <i class="fa fa-street-view"></i>
+                      <span
+                        >{{ client.address.Street }}, {{ client.address.City }},
+                        {{ client.address.State }}
+                      </span>
+                    </li>
+                    <li>
+                      <i class="fa fa-phone"></i>
+                      <span>{{ client.phone }}</span>
+                    </li>
+                    <li>
+                      <i class="fa fa-envelope"></i
+                      ><span>{{ client.email }}</span>
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <div class="card-text">
+              <p>
+                <b class="client-reviews">Reviews</b>
+
+                <span class="reviews">
```

Comment:
Is it possible to have a bootstrap `row` here and then have both anchor tags in their own `cols` to improve responsiveness? 

The cards are responsive but the two anchor tags get a bit off on a smaller screen.
<img width="559" alt="Screenshot 2019-09-02 at 12 39 51" src="https://user-images.githubusercontent.com/19375569/64105387-cf5c0b00-cd7e-11e9-9281-5689b9ff6818.png">

Reply:
@wangonya let me try that out.

Reply:
@wangonya I believe this looks better now, thanks to your recommendation.
<img width="729" alt="Screenshot 2019-09-02 at 13 05 03" src="https://user-images.githubusercontent.com/9926422/64107109-61b1de00-cd82-11e9-9b80-83ffe849b366.png">